stl/inc/compare

```diff
@@ -37,13 +37,6 @@ enum class _Compare_ncmp : _Compare_t { unordered = -128 };
 // CLASS partial_ordering
 class partial_ordering {
 public:

```

Comment:
I'm ever so mildly concerned that these are depicted as classes in the Standard given that MSVC name-mangles `class` and `struct` differently, but (a) I can't imagine it making difference except for the most contrived examples (virtual functions that take `partial_ordering` arguments??!?), and (b) I think we should do everything in our power to exacerbate that name-mangling bug so it may hopefully someday get fixed.